<launch>
      <arg name="id" default="fake0"/>
      <arg name="max_vel" default="3.0"/>
      <arg name="lat" default="0"/>
      <arg name="lon" default="8"/>
      <arg name="obstacle" default="True"/>
      <group ns="$(arg id)">
      <node pkg="robot_agent" type="fake_agent_node" name="fake_agent" output="screen" >
            <rosparam command="load" file="$(find robot_agent)/config/robot_agent_config.yaml" />
            <rosparam command="load" file="$(find robot_agent)/config/fake_agent_config.yaml" />
            <rosparam param="agent_id" subst_value="True">$(arg id)</rosparam>
            <rosparam param="fake_agent_max_vel" subst_value="True">$(arg max_vel)</rosparam>
            <rosparam param="fake_agent_initial_latitude" subst_value="True">$(arg lat)</rosparam>
            <rosparam param="fake_agent_initial_longitude" subst_value="True">$(arg lon)</rosparam>
            <rosparam param="obstacle_avoidance" subst_value="True">$(arg obstacle)</rosparam>
      </node>
      </group>
</launch>
